### DISCUSSION TITLE
How to express the x coordinate of the qp point in Kernel ?

### DISCUSSION THREAD

**Discussion by Ethan-xj** [idaholab/moose#22236] [Q&A General] [answered]
I know the `_q_point` is the coordinate of the current qp point. But how to take the x coordinate out?
I've tried to make `double xx = _q_point[0];`
But it report an error: cannot convert 'const libMesh::Point' to 'double' in initialization.

**Comment by GiudGiud** [comment, marked as answer]
_q_point is the array of the qps. So it needs to be double indexed into

`_q_point[qp](0)`
